zrml/neo-swaps/src/mock.rs

```diff
@@ -0,0 +1,545 @@
+    type MinOutcomeVoteAmount = MinOutcomeVoteAmount;
+    type RemoveKeysLimit = RemoveKeysLimit;
+    type GdVotingPeriod = GdVotingPeriod;
+    type VotingOutcomeFee = VotingOutcomeFee;
+    type WeightInfo = zrml_global_disputes::weights::WeightInfo<Runtime>;
+}
+
+impl zrml_swaps::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type ExitFee = ExitFee;
+    type FixedTypeU = <Runtime as zrml_rikiddo::Config>::FixedTypeU;
+    type FixedTypeS = <Runtime as zrml_rikiddo::Config>::FixedTypeS;
+    type LiquidityMining = LiquidityMining;
+    type MarketCommons = MarketCommons;
+    type MaxAssets = MaxAssets;
+    type MaxInRatio = MaxInRatio;
+    type MaxOutRatio = MaxOutRatio;
+    type MaxSwapFee = MaxSwapFee;
+    type MaxTotalWeight = MaxTotalWeight;
+    type MaxWeight = MaxWeight;
+    type MinAssets = MinAssets;
+    type MinSubsidy = MinSubsidy;
+    type MinSubsidyPerAccount = MinSubsidyPerAccount;
+    type MinWeight = MinWeight;
+    type PalletId = SwapsPalletId;
+    type RikiddoSigmoidFeeMarketEma = RikiddoSigmoidFeeMarketEma;
+    type AssetManager = AssetManager;
+    type WeightInfo = zrml_swaps::weights::WeightInfo<Runtime>;
+}
+
+impl pallet_treasury::Config for Runtime {
+    type ApproveOrigin = EnsureSignedBy<Sudo, AccountIdTest>;
+    type Burn = ();
+    type BurnDestination = ();
+    type Currency = Balances;
+    type RuntimeEvent = RuntimeEvent;
+    type MaxApprovals = MaxApprovals;
+    type OnSlash = ();
+    type PalletId = TreasuryPalletId;
+    type ProposalBond = ();
+    type ProposalBondMinimum = ();
+    type ProposalBondMaximum = ();
+    type RejectOrigin = EnsureSignedBy<Sudo, AccountIdTest>;
+    type SpendFunds = ();
+    type SpendOrigin = NeverEnsureOrigin<Balance>;
+    type SpendPeriod = ();
+    type WeightInfo = ();
+}
+
+#[cfg(feature = "parachain")]
+zrml_prediction_markets::impl_mock_registry! {
+    MockRegistry,
+    CurrencyId,
+    Balance,
+    zeitgeist_primitives::types::CustomMetadata
+}
+
+pub struct ExtBuilder {
+    balances: Vec<(AccountIdTest, Balance)>,
+}
+
+// FIXME Remove this in favor of adding whatever the account need in the individual tests.
+impl Default for ExtBuilder {
+    fn default() -> Self {
+        Self { balances: vec![(ALICE, 100 * BASE), (CHARLIE, BASE), (DAVE, BASE), (EVE, BASE)] }
+    }
+}
+
+impl ExtBuilder {
+    pub fn build(self) -> sp_io::TestExternalities {
+        let mut t = frame_system::GenesisConfig::default().build_storage::<Runtime>().unwrap();
+        pallet_balances::GenesisConfig::<Runtime> { balances: self.balances }
+            .assimilate_storage(&mut t)
+            .unwrap();
+        #[cfg(feature = "parachain")]
+        use frame_support::traits::GenesisBuild;
+        #[cfg(feature = "parachain")]
+        orml_tokens::GenesisConfig::<Runtime> {
+            balances: vec![(ALICE, FOREIGN_ASSET, _100)],
+            // balances: (0..69).map(|idx| (idx, FOREIGN_ASSET, _100)).collect(),
+        }
```

Comment:
Can we remove the comment?

```suggestion
        orml_tokens::GenesisConfig::<Runtime> {
            balances: vec![(ALICE, FOREIGN_ASSET, _100)],
        }
```